modules/renter/contractor/host_integration_test.go

```diff
@@ -104,9 +105,9 @@ func newTestingContractor(testdir string, cs modules.ConsensusSet, tp modules.Tr
 	return New(cs, w, tp, hdb, filepath.Join(testdir, "contractor"))
 }
 
-// newTestingTrio creates a Host, Contractor, and TestMiner that can be used
-// for testing host/renter interactions.
-func newTestingTrio(name string) (modules.Host, *Contractor, modules.TestMiner, error) {
+// newTestingTrio a Contractor, TestMiner, and a slice of hosts that can be
+// used for testing host/renter interactions.
+func newTestingTrio(name string, nhosts int) ([]modules.Host, *Contractor, modules.TestMiner, error) {
```

Comment:
I don't really like it; it's not a trio if there's multiple hosts. AFAICT, the only place we create multiple hosts is `TestIntegrationSetAllowance`, where we create 2 instead of 1. So I would prefer to revert the change, and instead either create the extra host manually or add a new helper function for creating multiple hosts.